platform/ios/src/MGLMapView.mm

```diff
@@ -750,6 +753,20 @@ - (void)glkView:(__unused GLKView *)view drawInRect:(__unused CGRect)rect
         _mbglMap->render();
 
         [self updateUserLocationAnnotationView];
+        
+        // Hide all annotation views
+        for (MGLAnnotationView *annotationView in _annotationViews) {
+            annotationView.hidden = YES;
+        }
+        
+        // Update visible annotation views
+        std::vector<MGLAnnotationTag> annotationTags = [self annotationTagsInRect:CGRectInset(self.bounds, 0, 0)];
```

Comment:
Understood. This is a WIP. I will be using something different than 0, 0 in a future commit. Please stay tuned.